rdr_service/api/nph_participant_api_schemas/schema.py

```diff
@@ -188,6 +190,19 @@ class Participant(ObjectType):
                                                  'completion of the SDOH module. Value should be UNSET or SUBMITTED '
                                                  'and time should be the authored time. Both should be sourced from '
                                                  'the AoU participant_summary table.')
+    # NPH
+    external_id = SortableField(String, name="nphPairedSite", description='Sourced from NPH Schema.',
+                                sort_modifier=lambda context: context.set_order_expression(nphSite.external_id))
+    organization_external_id = SortableField(String, name="nphPairedOrg", description='Sourced from NPH Schema.',
+                                             sort_modifier=lambda context: context.set_order_expression(
+                                                 nphSite.organization_external_id))
+    awardee_external_id = SortableField(String, name="nphPairedOrg", description='Sourced from NPH Schema.',
```

Comment:
This should be something like `nphPairedAwardee` not `nphPairedOrg`

Reply:
I'm being not careful about this one. already updated to show awardee_external_id to the name "nphPairedAwardee"